internal/clients/config/client.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+import (
+	"github.com/disaster37/go-kibana-rest/v8"
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients/fleet"
+)
+
+type Client struct {
```

Comment:
It contains configurations for clients. Shall we rename it to `ClientConfig`?

Reply:
It's within a `config` package, so usage would be `config.Client` `config.ClientConfig` is pretty stuttery IMO.